experimenter/experimenter/experiments/tests/api/v7/test_serializers.py

```diff
@@ -39,9 +39,9 @@ def test_serializer_outputs_expected_schema(self, localizations):
             secondary_outcomes=["quux", "xyzzy"],
             locales=[locale_en_us],
             is_localized=localizations is not None,
-            localizations=json.dumps(localizations)
-            if localizations is not None
-            else None,
+            localizations=(
+                json.dumps(localizations) if localizations is not None else None
```

Comment:
I think we can write this `localizations if localizations else None` or `localizations or None`

Reply:
Nope its a text field, not JSON